src/BenchmarksApps/Mvc/Program.cs

```diff
@@ -19,11 +23,21 @@ public static void Main(string[] args)
             CreateHostBuilder(args).Build().Run();
         }
 
-        public static IHostBuilder CreateHostBuilder(string[] args) =>
-            Host.CreateDefaultBuilder(args)
+        public static IHostBuilder CreateHostBuilder(string[] args)
+            => Host.CreateDefaultBuilder(args)
                 .ConfigureWebHostDefaults(webBuilder =>
                 {
-                    webBuilder.UseStartup<Startup>();
+                    webBuilder.UseStartup<Startup>()
+                        .ConfigureKestrel(options =>
+                        {
+#if CERTAUTH
+                            options.ConfigureHttpsDefaults(opt =>
+                            {
+                                opt.ClientCertificateMode = ClientCertificateMode.RequireCertificate;
+                                opt.AllowAnyClientCertificate();
```

Comment:
What about testing the one that we are using. I don't think that anyone using this feature would use Any certificate. Or do we know that the host path is still checked?

Reply:
For instance by passing the certificate file/url in an argument the same way it's done on the client

Reply:
Good point, I'll take a note for now to do a follow up PR that configures more complex validation to see how much that affects perf.